src/js/editing/Typing.js

```diff
@@ -97,14 +97,14 @@ export default class Typing {
 
           // replace empty heading, pre or custom-made styleTag with P tag
           if ((dom.isHeading(nextPara) || dom.isPre(nextPara) || dom.isCustomStyleTag(nextPara)) && dom.isEmpty(nextPara)) {
-            nextPara = dom.replace(nextPara, 'p');
+            nextPara = dom.replace(nextPara, singleSpace ? 'p' : 'div');
```

Comment:
I've noticed other ternaries in this diff do something with a `div` when evaluated to true; this one seems to do the opposite - just verifying that's intentional here

Reply:
Ah thanks, had 'em backwards there.